presto-main/src/main/resources/webapp/src/components/QueryDetail.jsx

```diff
@@ -1228,6 +1228,14 @@ export class QueryDetail extends React.Component {
                                     {query.queryStats.elapsedTime}
                                 </td>
                             </tr>
+                            <tr>
+                                <td className="info-title">
```

Comment:
did we test this if this shows up as expected in UI

Reply:
Yes - I did - anything that looks incorrect that prompted this question?

Reply:
Since we don't have any UI tests just wanted to make sure it was working :)